packages/server/lib/video_capture.coffee

```diff
@@ -18,6 +18,14 @@ debug("using ffmpeg from %s", ffmpegPath)
 
 ffmpeg.setFfmpegPath(ffmpegPath)
 
+maybeWriteChaptersMetaFile = (chaptersMetaFileName, ffmpegchaptersConfig) ->
+  new Promise (resolve, reject) ->
+    unless ffmpegchaptersConfig
+      return resolve(false)
+    fs.writeFileAsync chaptersMetaFileName, ffmpegchaptersConfig
+    .then () ->
+      resolve(true)
```

Comment:
You can use [`Promise.method`](http://bluebirdjs.com/docs/api/promise.method.html) instead of wrapping it manually:

```coffee
maybeWriteChaptersMetaFile = Promise.method (chaptersMetaFileName, ffmpegchaptersConfig) ->
  if not ...
    return false
  
  fs.writeFileAsync ...
  .return true
```